.github/workflows/docs/docs-test-and-lint.yml

```
@@ -0,0 +1,44 @@
+            - 'docs/**'
+    pull_request:
+        paths:
+            - 'docs/**'
+jobs:
+    test-docs:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout Repository
+                uses: actions/checkout@v4
+            -   name: Set Up Python
+                uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)